AccessibilitySnapshot/Classes/FBSnapshotTestCase+ObjCSupport.swift

```diff
@@ -49,6 +49,14 @@ extension FBSnapshotTestCase {
         activationPointDisplayMode: ActivationPointDisplayMode,
         useMonochromeSnapshot: Bool
     ) -> String? {
+        // The tests must be run in a host application in order for the accessibility properties to be populated
+        // correctly. The `UIApplication.shared` singleton is non-optional, but will be uninitialized when the tests are
+        // running outside of a host application, so we can use this check to determine whether we have a test host.
+        let hostApplication: UIApplication? = UIApplication.shared
+        guard hostApplication != nil else {
```

Comment:
Since we’re making this check more than once: thoughts on pulling out this check and associated comments into a method we can call in both places?

Reply:
Moved into a shared `isRunningInHostApplication` property 👍